tools/compatgen/main.go

```diff
@@ -30,7 +31,10 @@ func main() {
 					console.Fatalf("Failed to fetch CUDA base image tags: %s", err)
 				}
 			case "tensorflow":
-				v, err = internal.FetchTensorFlowCompatibilityMatrix()
+				if minCudaVersion == "" {
```

Comment:
This effectively makes a `--min-cuda-version` flag required. Instead, why don't we:

- Define a package-level `const defaultMinCudaVersion` set to 11.0 (or whatever). Add a comment with relevant date / link to provide context when that code is updated.
- Set `defaultMinCudaVersion` as the third argument to `StringVar(&minCudaVersion, ...` to set it as the default.
- Closer to the top of `RunE`, parse/validate `minCudaVersion` and return an error if it's an invalid version string.
- Later / Bonus: Update `FetchTorchCompatibilityMatrix` to take `minCudaVersion` argument.

Reply:
I was considering setting an minimum version here but I didn't want to leak the concern of what minimum version is acceptable from the `github.com/replicate/cog/pkg/config` package. But as there's a lot of coupling between the code here and `github.com/replicate/cog/pkg/config`, I am reconsidering this change and wondering if making this configurable via a flag is worth the expansion in the footprint.

I am proposing that we keep the FetchTensorFlowCompatibilityMatrix parameterized but just set it from a constant here.